Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -0,0 +1,259 @@
+import macro from 'vtk.js/Sources/macro';
+
+const MAX_POINTS = 2;
+
+export default function widgetBehavior(publicAPI, model) {
+  model.classHierarchy.push('vtkDistanceWidgetProp');
+  let isDragging = null;
+  const direction = [0, 0, 0];
+
+  // --------------------------------------------------------------------------
+  // Display 2D
+  // --------------------------------------------------------------------------
+
+  publicAPI.setDisplayCallback = (callback) =>
+    model.representations[0].setDisplayCallback(callback);
+
+  // --------------------------------------------------------------------------
+  // Interactor events
+  // --------------------------------------------------------------------------
+
+  function ignoreKey(e) {
+    return e.altKey || e.controlKey || e.shiftKey;
+  }
+
+  function updateConeDirection(behavior, callData) {
```

Comment:
When you call this function, you use numbers (1, 2, or 3). But it's quite difficult to understand how it correspond to until we go to the function. Maybe you can use an enum or something equivalent to be more explicit.